flink-connectors/flink-connector-elasticsearch6/pom.xml

```diff
@@ -197,29 +197,57 @@ under the License.
 								<configuration>
 									<shadedArtifactAttached>true</shadedArtifactAttached>
 									<shadedClassifierName>sql-jar</shadedClassifierName>
+									<artifactSet>
+										<includes>
+											<include>*:*</include>
+										</includes>
+										<excludes>
+											<!-- These dependencies are not required. -->
+											<exclude>com.carrotsearch:hppc</exclude>
+											<exclude>com.tdunning:t-digest</exclude>
+											<exclude>joda-time:joda-time</exclude>
+											<exclude>net.sf.jopt-simple:jopt-simple</exclude>
+											<exclude>org.elasticsearch:jna</exclude>
+											<exclude>org.hdrhistogram:HdrHistogram</exclude>
+											<exclude>org.yaml:snakeyaml</exclude>
+										</excludes>
+									</artifactSet>
 									<filters>
 										<filter>
-											<artifact>*:*</artifact>
-											<!-- It is difficult to find out artifacts that are really required by ES. -->
-											<!-- We use hard filters for now to clean up the SQL JAR. -->
+											<artifact>org.elasticsearch:elasticsearch</artifact>
 											<excludes>
-												<exclude>com/carrotsearch/**</exclude>
-												<exclude>com/sun/**</exclude>
```

Comment:
Which dependency pulls in this namespace?

Reply:
org.elasticsearch:jna